src/test/java/com/exasol/sql/expression/function/exasol/ExasolFunctionCastTest.java

```diff
@@ -10,18 +10,18 @@
 import com.exasol.sql.dql.select.Select;
 import com.exasol.sql.expression.NullLiteral;
 
-class ExasolCastFunctionTest {
-
+class ExasolFunctionCastTest {
     @Test
     void testRendering() {
-        final Select select = StatementFactory.getInstance().select().cast(NullLiteral.nullLiteral(), new Varchar(254));
+        final Select select = StatementFactory.getInstance().select()
```

Comment:
So you decided against refactoring it, to only test the ExasolFunctionCast API here and testing the select API somewhere else?

Reply:
It's your branch, you can do the refactoring. :)